app/src/main/java/org/mozilla/focus/home/pocket/Pocket.kt

```diff
@@ -120,3 +123,10 @@ data class PocketVideo(
         }
     }
 }
+
+fun ImageView.setImageDrawableAsPocketWordmark() {
+    // We want to set SVGs in code because they can produce artifacts otherwise.
```

Comment:
what does this mean, what are artifacts?

Reply:
An unintended visual anomaly generally caused by various implementation details. One example would be when you scale up an image and you can "see the pixels".

In this case, setting SVG in XML can distort it in arbitrary ways (like the ninja star substituting for the settings gear on the homescreen) so I've been blanket setting all SVGs in code.

See https://en.wikipedia.org/wiki/Visual_artifact for more.